uktrade/data-hub-frontend
# Update the global search header (#5191)

## Description of change

_Document what the PR does and why the change is needed_

## Test instructions

- Visit: http://localhost:3000/search/companies?term=
- You could see a link to Filter results which directs you to the relevant collection list 

## Screenshots

### Before

![image](https://user-images.githubusercontent.com/83657534/213246908-9c663f1e-bef0-4e0e-8469-a8fb07df5e8a.png)


### After

![image](https://user-images.githubusercontent.com/83657534/213247285-777fd004-3c76-42d3-9dd5-86af5a4e7d4b.png)



## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
